servicesExternal/web3_single.py

```diff
@@ -25,8 +25,8 @@ def getInstance():
     # We skip BN because it's useless in Python
 
     @staticmethod
-    def getNetworkName(networkId: int = 55) -> str:
-        return {1 : 'main', 2 :'morden', 3 : 'ropsten', 4 : 'rinkeby', 42 :'kovan',55 : 'private'}[int]
+    def getNetworkName(networkId: int ) -> str:
+        return {1 : 'main', 2 :'morden', 3 : 'ropsten', 4 : 'rinkeby', 42 :'kovan','default' : 'private'}.get(networkId,'default')
```

Comment:
The `.get(networkId, 'default') will return the string `'default'` if the `networkId` is not found. I think what you mean to do here is
```
return {1 : 'main', 2 :'morden', 3 : 'ropsten', 4 : 'rinkeby', 42 :'kovan'}.get(networkId, 'private')
```

Reply:
yes